### DISCUSSION TITLE
Request for feature (Completed )

### DISCUSSION THREAD

**Discussion by ramanveerji** [sachinsenal0x64/multi-gpt#53] [General]
Please add admin env so it can be accessed by admin or the users only whose id is present in admin env var.

**Comment by sachinsenal0x64** [comment]
ok i will add this asp

**Comment by sachinsenal0x64** [comment]
Added! https://github.com/sachinsenal0x64/MULTI-GPT/commit/cfb93ec83014d233b7fee988e10c0b421ab63be0
